inductiva/localization/translations/en.json

```
@@ -8,8 +8,52 @@
         "Unable to check for available updates for the 'inductiva' package.",
         "The following error occurred:\n  {0}"
     ],
-    "kill-request-sent":["Successfully sent kill request for task {0}.",
+    "task-request-sent":["Successfully sent kill request for task {0}.",
         "> Consider tracking the task status by calling task.get_status()",
         "> or via the CLI using 'inductiva tasks -id {0}'."
+    ],
+    "user-prompt-confirmation": [
+        "Are you sure you want to proceed (y/[N])? "
+    ],
```

single lined messages can be inlined (no need to be wrapped in an array):

```suggestion
    "user-prompt-confirmation": "Are you sure you want to proceed (y/[N])? ",
```